src/jit/ee_il_dll.cpp

```diff
@@ -163,6 +163,20 @@ CorJitResult CILJit::compileMethod (
         return g_realJitCompiler->compileMethod(compHnd, methodInfo, flags, entryAddress, nativeSizeOfCode);
     }
 
+    if (!JitConfig.isInitialized())
```

Comment:
> I'm guessing some obfuscated apps are not going to work. Unless the obfuscator passes a "real" VM ICorJitInfo* and not one they make themselves. Maybe that's already true.

I think it has to be true, otherwise any change we make to the JIT interface is going to break these apps.

> It seems like the "correct" place to put this would be as a new argument to getJit() and/or sxsJitStartup(), or to create a new "jitStartup()" api that would pass some ICorJitOneTimeStartup* interface, but I guess that would be more intrusive.

Yes, that's right. This is another place the inability to change the way the JIt is retrieved and initialized bites us.